_im-plugin/ism/settings.md

```diff
@@ -17,6 +17,7 @@ Setting | Default | Description
 :--- | :--- | :---
 `plugins.index_state_management.enabled` | True | Specifies whether ISM is enabled or not.
 `plugins.index_state_management.job_interval` | 5 minutes | The interval at which the managed index jobs are run.
+`plugins.index_state_management.jitter` | 0.6 | A delay that is added to a job's base run time to prevent a surge of activity from all indices at the same time. A value of 0.6 means 60% of run time is added to the base run time. For example, if you have a base run time of 30 minutes, a value of 0.6 means 18 minutes gets added to your run time, for a total of 48 minutes. Maximum is 1, which means an additional run time of 100%. This maximum cannot exceed `plugins.jobscheduler.jitter_limit`, which also has a default of 0.6. For example, if `plugins.index_state_management.jitter` is set to 0.8, ISM uses `plugins.jobscheduler.jitter_limit` of 0.6 instead.
```

Comment:
This isn't quite right. A value of 0.6 means that a random length of time between 0 and 60% of the interval period is added to the base interval. In the example with a job interval (not runtime) of 30 minutes, a value of 0.6 means that a random length of time between 0 and 18 minutes gets added to your job interval. 
Also, more generally, I think all of the uses of run time here should be job interval.

Reply:
Oops, that is an oversight on my part. Fixed to say 0-60% now.